src/run_assistant_audio.cc

```diff
@@ -216,26 +217,24 @@ int main(int argc, char** argv) {
     context.set_fail_fast(false);
     context.set_credentials(call_credentials);
 
-    std::shared_ptr<ClientReaderWriter<AssistRequest, AssistResponse>>
-        stream(std::move(assistant->Assist(&context)));
+    std::shared_ptr<ClientReaderWriter<AssistRequest, AssistResponse>> stream(
+        std::move(assistant->Assist(&context)));
     // Write config in first stream.
     if (verbose) {
       std::clog << "assistant_sdk wrote first request: "
                 << request.ShortDebugString() << std::endl;
     }
     stream->Write(request);
 
-      audio_input.reset(new AudioInputALSA());
+    audio_input.reset(new AudioInputALSA());
 
-      audio_input->AddDataListener(
+    audio_input->AddDataListener(
         [stream, &request](std::shared_ptr<std::vector<unsigned char>> data) {
           request.set_audio_in(&((*data)[0]), data->size());
           stream->Write(request);
-      });
-      audio_input->AddStopListener([stream]() {
-        stream->WritesDone();
-      });
-      audio_input->Start();
+        });
+    audio_input->AddStopListener([stream]() { stream->WritesDone(); });
```

Comment:
Is this the right format, to put everything on one line?

Reply:
It seems so if the lambda parameter is a one liner.
If I put a second line inside the lambda it gets splitted on two lines, so my understanding is that it's the way the Google's style works in this case.